src/main/java/launutsapi/LaunutsController.java

```diff
@@ -1,8 +1,155 @@
 package launutsapi;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.simple.JSONArray;
+import org.json.simple.parser.ParseException;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 @RestController
 public class LaunutsController {
+	
+	@Autowired
+	private NutsService nut_service;
+	
+	@Autowired
+	private LauService lau_service;
+	
+	private Nuts nut;
+	private Lau lau;
+	
+	private String notification = "Please enter query in correct format."
+		     + "Some correct formats are like these "
+		     + "http://localhost:8080/launuts/nuts/DEA47/json, "
+		     + "http://localhost:8080/launuts/nuts/DEA47/ttl, "
+		     + "http://localhost:8080/launuts/nuts/paderborn/json, "
+		     + "http://localhost:8080/launuts/nuts/paderborn/ttl, "
+		     + "http://localhost:8080/launuts/nuts/ttl, "
+		     + "http://localhost:8080/launuts/nuts/json ";
+	
+	@RequestMapping("/launuts/nuts/{query_string}/json")
+	public List<Nuts> getNutsJson(@PathVariable String query_string) throws IOException, ParseException {
+		
+		//Validate input: If user trying to enter a nut id
+		if(query_string.matches(".*\\d.*") && !(query_string.toLowerCase().matches("^((de)|(de\\D))((\\d{1})|(\\d{2})|(\\d{3}))$"))) {
+	
+			nut = new Nuts(notification);	
+			return Arrays.asList(nut);	
+		}
+		else 
+			return Arrays.asList(nut_service.getNutsJson(query_string));	
+			
+	}
+	
+	@RequestMapping("/launuts/lau/{query_string}/json")
```

Comment:
Same case when we write location name. I have attached an image below for reference.
![error3](https://user-images.githubusercontent.com/52550720/84090987-81ea8980-a9f3-11ea-90aa-e5af0dd72da7.JPG)

Reply:
Yes, It was part of the same bug. It is resolved as well. Thanks for the review.

Reply:
It is working fine now, good.